app/steps/ui/shutterpage/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Step = require('app/core/steps/Step');
+
+module.exports = class ShutterPage extends Step {
+
+    static getUrl() {
+        return '/shutter-page';
+    }
```

Comment:
Can we have a unit test for this method?

See `test/unit/testMentalCapacity.js` for an example